docs/modules/qbi-helper/interest-template.md

```diff
@@ -0,0 +1,141 @@
+<%(TOC)%>
+
+# How to use InterestTemplate
+
+<%(Callout message="
+Before reading this document, make sure you have read:
+
+  * [Introduction to Query-based interest]({{urlRoot}}/modules/qbi-helper/intro-to-qbi)
+")%>
+
+An [`InterestTemplate`]({{urlRoot}}/api/query-based-interest/interest-template) is effectively a wrapper around the `Interest` component, providing ways to more easily add, replace and clear queries from the underlying interest map.
+
+## Initialise
+
+You initialise an empty [`InterestTemplate`]({{urlRoot}}/api/query-based-interest/interest-template) by calling `InterestTemplate.Create()`. This is useful when defining a set of interest queries for the first time, for example when defining Entity Templates.
+
+```csharp
+var basicInterestTemplate = InterestTemplate.Create();
+```
+
+You can also construct this from an existing [`InterestTemplate`]({{urlRoot}}/api/query-based-interest/interest-template) or set of interest queries. This creates a deep copy of the [`InterestTemplate`]({{urlRoot}}/api/query-based-interest/interest-template), to allow you to modify the queries without affecting the original set.
+
+```csharp
+var advancedInterestTemplate = InterestTemplate.Create(basicInterestTemplate);
+```
+
+## Modify
+
+When modifying an [`InterestTemplate`]({{urlRoot}}/api/query-based-interest/interest-template), you must specify which particular component's queries you are modifying. You can either:
+
+* provide the component as a type argument
```

Comment:
Type argument or generic argument? Not sure what the standard is

Reply:
If we were talking about it in the method, I'd call it a `generic type parameter` - so perhaps here we should cal it a `generic type argument`?

Reply:
calling it a `generic type parameter`